add.py

```diff
@@ -1,6 +1,18 @@
+def calculate_average(numbers):
+    """
+    Calculate the average of a list of numbers.
+    """
+    t = 0
```

Comment:
You use t at the beginning but increment total inside the loop though it is undeclared. Please fix